Header header
string obstacle_data
string door_status
